### DISCUSSION TITLE
jbrowse-react-linear-genome-view: Multiple Assembies

### DISCUSSION THREAD

**Discussion by arpanda** [GMOD/jbrowse-components#2436] [Q&A]
Is it possible to add multiple assemblies in [jbrowse-react-linear-genome-view](https://github.com/GMOD/jbrowse-components/tree/main/products/jbrowse-react-linear-genome-view) ? I tried to add multiple assemblies by modifying the *assembly* section in the below code structure, but it's showing an error. Probably that assembly section takes only one assembly, so is there any way to load multiple assemblies. 

```
import React from 'react'
import 'fontsource-roboto'
import {
  createViewState,
  JBrowseLinearGenomeView,
} from '@jbrowse/react-linear-genome-view'

function View() {
  const state = createViewState({
    assembly: [{
      /* assembly 1*/
    },
{
  /* assembly 2*/
}
 ],
    tracks: [
      /* tracks */
    ],
  })
  return <JBrowseLinearGenomeView viewState={state} />
}


```

**Comment by cmdcolin** [comment]
Currently it is not supported to do multiple assemblies. If you want to switch assemblies, we suggest re-creating the view state e.g. rerunning createViewState when an assembly changes. If you have a need to display multiple assemblies at once let us know, and we can consider this though.

**Reply by arpanda** [reply]
Thanks.
Currently, I can re-create the view state for each assembly. Although, the multiple assemblies option will be helpful.
